modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/internal/instance/lifecycle/test/DepotRolesPortalInstanceLifecycleListenerTest.java

```diff
@@ -41,6 +46,7 @@ public class DepotRolesPortalInstanceLifecycleListenerTest {
 			new LiferayIntegrationTestRule(),
 			PermissionCheckerMethodTestRule.INSTANCE);
 
+	@FeatureFlags("LPD-17564")
```

Comment:
I do not like to "lose" a test without the FF, but ok

Reply:
@AliciaGarciaGarcia, I know, it's temporary, still, the code will still execute :D